File: result/translate/starchat/python2java/java/SORT_EVEN_PLACED_ELEMENTS_INCREASING_ODD_PLACED_DECREASING_ORDER.java
public static void f_gold ( int arr [ ], int n ) {
        int evenArr [ ] = new int [ n ];
        int oddArr [ ] = new int [ n ];
        for ( int i = 0 ; i < n ; i ++ ) {
            if ( ( i % 2 ) == 0 ) {
                evenArr [ ] = arr [ i ];
            } else {
                oddArr [ ] = arr [ i ];
            }
        }
        Arrays.sort ( evenArr );
        Arrays.sort ( oddArr, Collections.reverseOrder ( ) );
        int i = 0;
        for ( int j = 0 ; j < evenArr.length ; j ++ ) {
            arr [ i ] = evenArr [ j ];
            i ++;
        }
        for ( int j = 0 ; j < oddArr.length ; j ++ ) {
            arr [ i ] = oddArr [ j ];
            i ++;
        }
    }

File: result/translate/transcoder/java2python/python/SORT_EVEN_PLACED_ELEMENTS_INCREASING_ODD_PLACED_DECREASING_ORDER.py
def f_gold ( arr , n ) :
    evenArr = [ ]
    oddArr = [ ]
    for i in range ( n ) :
        if i % 2 != 1 :
            evenArr.append ( arr [ i ] )
        else :
            oddArr.append ( arr [ i ] )
    evenArr.sort ( )
    oddArr.sort ( )
    i = 0
    for j in evenArr :
        arr [ i ] = evenArr [ j ]
    for j in oddArr :
        arr [ i ] = oddArr [ j ]
    return arr